ensime_shared/launcher.py

```diff
@@ -163,6 +163,13 @@ def _start_process(self, classpath):
         null = open(os.devnull, "r")
         java = os.path.join(self.config['java-home'], 'bin', 'java')
 
+        if(os.name == "nt"):
+            java += ".exe" #the java executable has the .exe extension in windows
+            # windows uses ';' instead of ':' for the classpath sep since ':' is reserved (C:\)
+            # for example  "c:\ensime.jar:c:\scala.jar"  =>  "c:\ensime.jar;c:\scala.jar"
+            # the following ".jar:" hack works because only jar's happen to be on the ensime classpath
+            classpath = classpath.replace(".jar:",".jar;")
```

Comment:
Ok, I pushed the changes (see 1a86e7128f01afa6adc25578223941cb184c2456) to make the classpath a list and only make it into a string at the argument location. 

I was also able to successfully run the `make test` after changing the classpath there to lists also.   However, in order to run the tests on windows, I had to change the Makefile but kept this [in a different branch](/drxcc/ensime-vim/windows-test).